impl/src/main/java/org/jboss/forge/roaster/model/util/Formatter.java

```diff
@@ -191,33 +204,18 @@ private static Properties readConfigInternal()
       return properties;
    }
 
-   private static Properties parseConfig(InputStream stream)
-            throws IOException
+   private static Properties parseConfig(InputStream stream) throws IOException
    {
-      try
-      {
-         final Properties formatterOptions = readConfigInternal();
-         formatterOptions.load(stream);
-         return formatterOptions;
-      }
-      finally
-      {
-         Streams.closeQuietly(stream);
-      }
+      final Properties formatterOptions = readConfigInternal();
+      formatterOptions.load(stream);
```

Comment:
You need to close this stream, no?

Reply:
No, as defined in [The try-with-resources Statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html): "The try-with-resources statement ensures that each resource is closed at the end of the statement".

Reply:
No, the stream comes from a try-with-resource statement and so it will be automatic be closed